09-features.md

```diff
@@ -19,40 +19,43 @@ for use of the channel, so the presentation of those features depends
+| 0/1   | `option_data_loss_protect`        | Requires or supports extra `channel_reestablish` fields  | IN       |                           | [BOLT #2][bolt02-retransmit]                                         |
+| 3     | `initial_routing_sync`            | Sending node needs a complete routing information dump   | I        |                           | [BOLT #7][bolt07-sync]                                               |
+| 4/5   | `option_upfront_shutdown_script`  | Commits to a shutdown scriptpubkey when opening channel  | IN       |                           | [BOLT #2][bolt02-open]                                               |
+| 6/7   | `gossip_queries`                  | More sophisticated gossip control                        | IN       |                           | [BOLT #7][bolt07-query]                                              |
+| 8/9   | `var_onion_optin`                 | Requires/supports variable-length routing onion payloads | IN9      |                           | [Routing Onion Specification][bolt04]                                |
+| 10/11 | `gossip_queries_ex`               | Gossip queries can include additional information        | IN       | `gossip_queries`          | [BOLT #7][bolt07-query]                                              |
+| 12/13 | `option_static_remotekey`         | Static key for remote output                             | IN       |                           | [BOLT #3](03-transactions.md)                                        |
+| 14/15 | `payment_secret`                  | Node supports `payment_secret` field                     | IN9      | `var_onion_optin`         | [Routing Onion Specification][bolt04]                                |
+| 16/17 | `basic_mpp`                       | Node can receive basic multi-part payments               | IN9      | `payment_secret`          | [BOLT #4][bolt04-mpp]                                                |
+| 18/19 | `option_support_large_channel`    | Can create large channels                                | IN       |                           | [BOLT #2](02-peer-protocol.md#the-open_channel-message)              |
+| 20/21 | `option_anchor_outputs`           | Anchor outputs                                           | IN       | `option_static_remotekey` | [BOLT #3](03-transactions.md)                                        |
+| 22/23 | `option_anchors_zero_fee_htlc_tx` | Anchor commitment type with zero fee HTLC transactions   | IN       |                           | [BOLT #3][bolt03-htlc-tx], [lightning-dev][ml-sighash-single-harmful]|
+| 24/25 | `option_compression`              | Compression algorithms advertised in `init`              | IN       |                           | [BOLT #1](01-messaging.md#the-init-message)                          |
```

Comment:
Electrum version 4.1.0 uses bit 25 for Trampoline: https://github.com/spesmilo/electrum/blob/92226b077ad9d248e0f3325f3de89b34b1778ea5/RELEASE-NOTES#L76

Reply:
That sounds...risky. There's no guarantee this will be the final spec feature bit.